tutorials/jmsActiveMQ/src/site/conf/my-app-conf1.xml

```diff
@@ -1,11 +1,28 @@
 <?xml version="1.0" encoding="UTF-8" standalone="no"?>
 <configuration>
-  <property>
-    <name>dt.attr.MASTER_MEMORY_MB</name>
-    <value>1024</value>
-  </property>
-  <property>
-    <name>dt.application.MyFirstApplication.operator.randomGenerator.prop.numTuples</name>
-    <value>1000</value>
-  </property>
-</configuration>
+	<property>
+		<name>dt.operator.fileOut.prop.filePath</name>
+		<value>target/</value>
```

Comment:
Since this file is intended to be used on a cluster, shouldn't the value be a full  HDFS path like "/tmp" ? Does this value actually work ?